kludges.js

```diff
@@ -527,7 +331,7 @@ function enable() {
     // Don't hide notifications when there's fullscreen windows in the workspace.
     // Fullscreen windows aren't special in paperWM and might not even be
     // visible, so hiding notifications makes no sense.
-    with (imports.ui.messageTray) {
+    with (MessageTray) {
```

Comment:
Also what does this `with` do? I have never seen that in JS. According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/with it should be avoided.

Reply:
This is a good question!  I'm not sure - I just changed the imports reference.  Let's look at changing it.

Reply:
So! turns out this monkey patching of MessageTray is actually not needed at all (should have testing just removing this for #553).  That is, default gnome behaviour now actually does what #553 was doing by modifying the monkey patch!

Removed it entirely.  Problem solved!